### PR TITLE
dark-mode feature

### LINKED ISSUE #14: Add DarkTheme and Light Theme feature for the App
The colour scheme should toggle between light and theme designs and option for it will be in menu option.

### PR DESCRIPTION
**Pull request by codingninja11**
closes #14 

https://user-images.githubusercontent.com/64359203/124494405-6ad3c200-ddd4-11eb-9601-c2bd2cd2a30a.mp4

@soumik2012

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import './App.css';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (soumikbaithalu) [inline src/App.js:2]: remove this use Effect  and use state

**Review by soumikbaithalu — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (soumikbaithalu) [reply src/App.js:2]: image

On the login page we clicking on the dark mode button it is not changing the dark color or please change the size of the button to normal.

Comment (soumikbaithalu) [reply src/App.js:2]: image

Add the dark button in here.  or please remove all the errors and warnings then make the PR @codingninja11

Comment (soumikbaithalu) [reply src/App.js:2]: If are done, then feel free to Re-request review

Comment (codingninja11) [reply src/App.js:2]: > remove this use Effect  and use state 

why it should be removed?please explain!
@soumik2012

Comment (codingninja11) [reply src/App.js:2]: > image
> 
> Add the dark button in here.  or please remove all the errors and warnings then make the PR @codingninja11

There are no errors when I'm running it on my device and for adding the functionality here please guide me how can I implement the same thing here. 
@soumik2012

Comment (soumikbaithalu) [reply src/App.js:2]: > > remove this use Effect  and use state
> 
> why it should be removed?please explain!
> @soumik2012

image
see this error also it's not deploying it giving the fail
image

**Comment by soumikbaithalu** [comment]
Closing the issue due to no response Thank you for making the PR you can check other issue

**Pull request closed without merging**